packages/dds/tree/src/shared-tree-core/branch.ts

```diff
@@ -594,20 +592,22 @@ export class SharedTreeBranch<
 }
 
 /**
- * Registers an event listener that fires when the given forkable object forks.
+ * Registers an event listener that fires when the given branch forks.
  * The listener will also fire when any of those forks fork, and when those forks of forks fork, and so on.
```

Comment:
I'm not sure if you want to update this line of text as well.

Reply:
Nah, it's okay to call a "branch" a "fork" (in this case that means it was made via `fork()` rather than just being created outright). I just wanted to get rid of the now-underspecificity of "forkable object".